packages/scene-composer/src/components/panels/SceneHierarchyPanel/components/SceneHierarchyTree/__tests__/__snapshots__/SceneHierarchyTreeItem.spec.tsx.snap

```diff
@@ -23,3 +23,15 @@ exports[`SceneHierarchyTreeItem should render SubModelTree when item has a model
   />
 </div>
 `;
+
+exports[`SceneHierarchyTreeItem should render dynamic node with drag and drop disabled 1`] = `
+<div>
+  <enhancedtreeitem
+    acceptdrop=""
+    data="[object Object]"
```

Comment:
Can we update these to reflect more meaningful data rather than `[object Object]`?

Reply:
updated